app/controllers/api/v1/statistics_controller.rb

```diff
@@ -10,7 +10,8 @@ def index
           number_of_students: number_of_students,
           number_of_mentors: number_of_mentors,
           duration_of_project_in_weeks: duration_of_project_in_weeks,
-          number_of_completed_questionnaires: number_of_completed_questionnaires([Person::STUDENT, Person::MENTOR])
+          number_of_completed_questionnaires: number_of_completed_questionnaires([Person::STUDENT, Person::MENTOR]),
+          number_of_book_signups: number_of_book_signups
```

Comment:
het is ok dat dit publiek is? die statistics api is (by design) zonder auth

Reply:
Ja lijkt me niet echt een privacy-sensitive gegeven verder. Het is net als een teller op een pagina.